src/context/AppContext.js

```diff
@@ -9,37 +21,118 @@ export const useApp = () => {
 }
 
 const AppProvider = ({children}) => {
-  // global useState
-  const tourData = useQuery(TOURS_QUERY)
-  const [tours, setTours] = useState(tourData.data ? tourData.data.tours : [])
+  // GLOBAL STATE
+  // intial data -> state match queries
+  const [tours, setTours] = useState([])
+  const [events, setEvents] = useState()
+  const [venues, setVenues] = useState([])
+  const [contacts, setContacts] = useState([])
+  const [organization, setOrganization] = useState()
+  const [user, setUser] = useState()
 
-  const eventsData = useQuery(EVENTS_QUERY)
-  const [events, setEvents] = useState(eventsData.data ? eventsData.data.events : [])
-  console.log(eventsData)
+  // queries
+  useQuery(TOURS_QUERY, {
+    onCompleted: data => {
+      setTours(data.tours)
+      setLoading(false)
+      setError(false)
+    }
+  })
 
-  const [contacts, setContacts] = useState([])
+  useQuery(EVENTS_QUERY, {
+    onCompleted: data => {
+      setEvents(data.events)
+    }
+  })
+
+  // useQuery(CONTACTS_QUERY, {
+  //   onCompleted: data => {
+  //     setContacts(data.contacts)
+  //   }
+  // })
+
+  useQuery(VENUE_QUERY, {
+    onCompleted: data => {
+      setVenues(data.venues)
+    }
+  })
+
+  useQuery(ORGANIZATION_QUERY, {
+    onCompleted: data => {
+      console.log(data)
+      setOrganization(data.organization)
+    },
+    onError: error => {
+      console.log(error)
+    }
+  })
+
+  useQuery(USER_QUERY, {
+    onCompleted: data => {
+      setUser(data.user)
+    }
+  })
+  
+  // mutations
+    // const EVENTS_MUTATION = gql`
+    //   mutation {
+    //     createEvent(input: {
+    //       tourId: 1,
+    //       name: "TEST",
+    //       venueId: 3,
+    //       startTime: "2021-04-22",
+    //       endTime: "2021-04-22"
+    //     })
+    //     {
+    //       name
+    //     }
+    //  `
+  // const [ helperFunuction ] = useMutation(EVENTS_MUTATION)
 
+  // generic state info
   const [error, setError] = useState('')
   const [loading, setLoading] = useState('')
   const [message, setMessage] = useState('')
 
-  // functions
+  // FUNCTIONS
+  const updateTours = (newTour) => {
+    setTours([...tours, newTour])
+    //setLoading, setError
+    //ADD MUTATIONS TO update
+    //resetLoading, resetError
+  }
+
+  const updateEvents = (newEvent) => {
+    setEvents([...events, newEvent])
+    // helperFunction({input: newEvent})
+  }
+
+  const updateVenues = (newVenue) => {
+    setVenues([...venues, newVenue])
+    
+    //setLoading, setError
+    //ADD MUTATIONS
+    //resetLoading, resetError
+  }
+
+  const updateContacts = (newContact) => {
+    setContacts([...contacts, newContact])
+    //setLoading, setError
+    //ADD MUTATIONS
+    //resetLoading, resetError
+  }
 
-  // const getEvents = () => {
-    // do API call
-    // apiCall.getEvents
-    // return what we want
-  // }
 
-  // values
+  // VALUES
```

Comment:
I feel like I've seen this, but is all caps the common practice for comments? Good catch. I just had no idea.